ddtrace/bootstrap/sitecustomize.py

```diff
@@ -81,6 +97,9 @@ def update_patched_modules():
     if 'DATADOG_ENV' in os.environ:
         tracer.set_tags({"env": os.environ["DATADOG_ENV"]})
 
+    if 'DD_TRACE_TAGS' in os.environ:
```

Comment:
Just a note for future reference when we add support for the helper to extract environment variables that we should convert these to the helper. It's fine to leave as-is for now.

Reply:
Noted!